create-book-client/src/main/kotlin/library/createbookclient/CreateBookClientApplication.kt

```diff
@@ -32,12 +33,13 @@ class CreateBookClientApplication(
             title = inputValues.first()
         )
         runBlocking {
-            createBookConsumer.sendBook(
-                isbn = bookRequest.isbn,
-                title = bookRequest.title
-            )
+            launch {
```

Comment:
What's the point of launching of coroutine here if we immediately block to wait for it?